services/ui_backend_service/data/cache/generate_dag_action.py

```diff
@@ -86,8 +87,17 @@ def execute(cls,
         result_key = [key for key in keys if key.startswith('dag:result')][0]
 
         with streamed_errors(stream_output):
+            # TODO: Clean up the internal _graph_info accessing.
+            # Having to jump through a lot of hoops currently due to internal artifacts
```

Comment:
easiest change is if a new client version provides a property on the run, along with fallback logic that parses graph_info from codepackage if the artifact is missing.